packages/hub/src/exports.ts

```diff
@@ -78,16 +51,16 @@ export function captureMessage(
  * @param event The event to send to Sentry.
  * @returns The generated eventId.
  */
-export function captureEvent(event: Event): string {
-  return callOnHub('captureEvent', event);
+export function captureEvent(event: Event): ReturnType<Hub['captureEvent']> {
+  return getCurrentHub().captureEvent(event);
 }
 
 /**
  * Callback to set context information onto the scope.
  * @param callback Callback function that receives Scope.
  */
-export function configureScope(callback: (scope: Scope) => void): void {
-  callOnHub<void>('configureScope', callback);
+export function configureScope(callback: (scope: Scope) => void): ReturnType<Hub['configureScope']> {
+  getCurrentHub().configureScope(callback);
```

Comment:
Agree with the `return`, but I'd leave the type as is. This way we are 100% sure that they are in sync.

Reply:
So I initially had the return types (see: https://github.com/getsentry/sentry-javascript/pull/4971/commits/3d3dd2b2dc8ee99c499d0607a6a000ef8d272148), but I removed because it's unnecessary bundle size (can't minify return). I can add a note about this.

Will add migration docs!